index.html

```diff
@@ -9,56 +9,52 @@
 
   <body>
     <header>
-
       <nav>
-       <ul class="left">
-         <li><img class="shoelogo" src="logo.jpg" alt="shoelogo"></li>
-         <li><a href="#">MEN</a></li>
-         <li><a href="#">WOMEN</a></li>
-         <li><a href="#">BOYS</a></li>
-         <li><a href="#">GIRLS</a></li>
-         <li><a href="#">CUSTOMIZE</a></li>
-       </ul>
-       <ul class="right">
-         <li><a href="#">HELP</a></li>
-         <li><a href="#">MY ACCOUNT</a></li>
-         <li><a href="#"><img src="cart.jpg" alt="shopping cart logo">shopping cart (2)</a></li>
-      </ul>
+        <ul class="left">
+           <li><img class="shoelogo" src="logo.jpg" alt="shoelogo"></li>
+           <li><a href="#">MEN</a></li>
+           <li><a href="#">WOMEN</a></li>
+           <li><a href="#">BOYS</a></li>
+           <li><a href="#">GIRLS</a></li>
+           <li><a href="#">CUSTOMIZE</a></li>
+         </ul>
+
+         <ul class="right">
+           <li><a href="#">HELP</a></li>
+           <li><a href="#">MY ACCOUNT</a></li>
+           <li><a href="#"><img src="cart.jpg" alt="shopping cart logo">shopping cart (2)</a></li>
+        </ul>
+      </nav>
     </header>
 
-      <p class="shoe-background"><img src="shoe1.png" alt="picture of black and white sneaker" class="shoe1"></p>
-      <ul>
-
-      <p class="shoe-background"><img src="shoe2.png" alt="picture of blue and orange sneaker" class="shoe2">
-        <img src="shoe3.png" alt="picture of red and black sneaker"  class="shoe3">
-        <img src="shoe4.png" alt="picture of blue and white sneaker" class="shoe4">
+    <p class="shoe-background tall">
+      <img src="shoe1.png" alt="picture of black and white sneaker" class="shoe1">
+    </p>
 
+    <div class="shoe-background short">
```

Comment:
We didn't go too deep into this in class, but there's an important detail here: you can't put a `<ul>` within a `<p>`. Instead, I swapped this out for a `<div>`, which allows any sort of content. This lets you stack the shoe images within your `<ul>` on top of the background.